docs/en_us/enrollment_api/source/authentication.rst

```diff
@@ -0,0 +1,23 @@
+.. _edX API Authentication:
+
+#############################
+edX API Authentication
+#############################
+
+
+*************
+OAuth 2.0
```

Comment:
There is also session based authentication.  As mentioned above, the course endpoint (just getting course enrollment information) does not require authentication, so users can find course enrollment details without an account.